docs/topics/mta-web-assessing-apps.adoc

```diff
@@ -6,6 +6,10 @@
 [id="mta-web-assessing-apps_{context}"]
 = Assessing an application by using the MTA assessment questionnaire
 
+The assessment process for applications that are to be migrated in {ProductFullName} has been separated from other features. In the {ProductShortName} user interface, you can assess an application and display the currently saved assessments by using the Assessment module.
+
+The {ProductShortName} assesses applications according to a set of questions relevant to the application, such as dependencies.
```

Comment:
I don't know if that belongs here but the user can also upload their own questionnaires

Reply:
Hmmm... @abrugaro, does the procedure differ somehow if the user uses their own questionnaires? If so, I'd create a different procedure..

Reply:
The procedure for assessing an application is the same, no matter if its the system's questionnaire or a custom one.

Reply:
@abrugaro, I can see that the sentence right below mentions the usage of Legacy Pathfinder...So, since this procedure is the same, I suggest rewriting the sentence below to something like "To assess the application, you can use both the default Legacy Pathfinder MTA questionnaire and your custom questionnaire." What do you think about this kind of wording?

Reply:
Sounds good to me, I may change `questionnaire` for `questionnaires` as an user can import and use multiple custom questionnaires